src/components/table/table.js

```diff
@@ -281,8 +295,9 @@ class Table extends PureComponent {
             )
         }
         <div
+          id="tableWrapper"
```

Comment:
better to use `ref`

Reply:
As we can have more table components in the document

Reply:
https://www.javascriptstuff.com/use-refs-not-ids/ --> nice explanation with a demo